File: AppGenerateFiles/dbo/FrontModel/Estudiante_Clases_ViewViewModel.js
//@ts-check
import { EntityClass } from "../../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools, BasicStates } from "../../../WDevCore/WModules/WComponentsTools.js";
//@ts-ignore
import { ModelProperty } from "../../../WDevCore/WModules/CommonModel.js";
class Estudiante_Clases_View extends EntityClass {
   /** @param {Partial<Estudiante_Clases_View>} [props] */
   constructor(props) {
       super(props, 'ViewDbo');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Date}*/ Transferido;
   /**@type {Number}*/ Estudiante_id;
   /**@type {Number}*/ Id;
   /**@type {Date}*/ Retirado;
   /**@type {Number}*/ Promedio;
   /**@type {Boolean}*/ Repitente;
   /**@type {Number}*/ Reprobadas;
   /**@type {String}*/ Nombre_periodo;
   /**@type {String}*/ Nombre_corto_periodo;
   /**@type {Date}*/ Inicio_periodo;
   /**@type {Date}*/ Fin_periodo;
   /**@type {Boolean}*/ Abierto;
   /**@type {Boolean}*/ Oculto;
   /**@type {String}*/ Nombre_nota;
   /**@type {String}*/ Nombre_corto_nota;
   /**@type {Number}*/ Numero_consolidados;
   /**@type {Number}*/ Consolidado_id;
   /**@type {Number}*/ Orden;
   /**@type {Number}*/ Resultado;
   /**@type {String}*/ Tipo;
   /**@type {}*/ Hora;
   /**@type {Date}*/ Fecha;
   /**@type {Number}*/ Porcentaje;
   /**@type {String}*/ Nombre_asignatura;
   /**@type {String}*/ Nombre_corto_asignatura;
   /**@type {String}*/ Nombre_grado;
   /**@type {String}*/ Nombre_corto_nivel;
   /**@type {String}*/ Nombre_nivel;
   /**@type {Number}*/ Numero_grados;
   /**@type {Number}*/ Inicio_grado;
   /**@type {Number}*/ Grado;
   /**@type {Number}*/ Clase_id;
}
export { Estudiante_Clases_View }

File: AppGenerateFiles/dbo/Model/Estudiante_Clases_ViewViewModel.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class Estudiante_Clases_View : EntityClass {
       public DateTime? Transferido { get; set; }
       public int? Estudiante_id { get; set; }
       public int? Id { get; set; }
       public DateTime? Retirado { get; set; }
       public Double? Promedio { get; set; }
       public bool? Repitente { get; set; }
       public int? Reprobadas { get; set; }
       public string? Nombre_periodo { get; set; }
       public string? Nombre_corto_periodo { get; set; }
       public DateTime? Inicio_periodo { get; set; }
       public DateTime? Fin_periodo { get; set; }
       public bool? Abierto { get; set; }
       public bool? Oculto { get; set; }
       public string? Nombre_nota { get; set; }
       public string? Nombre_corto_nota { get; set; }
       public int? Numero_consolidados { get; set; }
       public int? Consolidado_id { get; set; }
       public int? Orden { get; set; }
       public Double? Resultado { get; set; }
       public string? Tipo { get; set; }
       public ? Hora { get; set; }
       public DateTime? Fecha { get; set; }
       public Double? Porcentaje { get; set; }
       public string? Nombre_asignatura { get; set; }
       public string? Nombre_corto_asignatura { get; set; }
       public string? Nombre_grado { get; set; }
       public string? Nombre_corto_nivel { get; set; }
       public string? Nombre_nivel { get; set; }
       public int? Numero_grados { get; set; }
       public int? Inicio_grado { get; set; }
       public int? Grado { get; set; }
       public int? Clase_id { get; set; }
   }
}
